src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -2512,6 +2512,10 @@ public DescribeResults describe(String uuidX, DescribeQuery hello) {
    */
   public void onDescribe(DescribeResults results) {
     List<Registration> reservations = results.getReservations();
+    if (getId().equals("c1"))

```

Comment:
What does "c1" represent here? Is this debug code that should be removed?

Reply:
yeah - should be removed - but not ready yet ...
I want **_describe_** to work similarly to kubernetes describe - will take some more effort